user-docs.md

```diff
@@ -0,0 +1,99 @@
+# Algo Orders Description
+
+## TOC
+
+ - [Accumulate/Distribute](#ad)
+ - [Iceberg](#iceberg)
+ - [MA Crossover](#macross)
+ - [Order Creates OCO](#ococo)
+ - [Ping/Pong](#pingpong)
+ - [TWAP](#twap)
+
+## Accumulate/Distribute
+
+<a id="ad" />
+Accumulate/Distribute allows you to break up a large order into smaller
+randomized chunks, submitted at regular or irregular intervals to minimise
+detection by other players in the market.
+<br/>
+<br/>
+By enabling the 'Await Fill' option, the algorithm will ensure each
+component fills before submitting subsequent orders. Enabling the 'Catch Up'
+flag will cause the algorithm to ignore the slice interval for the next order
+if previous orders have taken longer than expected to fill, thereby ensuring
+the time-to-fill for the entire order is not adversely affected.
+
+The price must be manually specified as `limitPrice` for `LIMIT` order types,
+or as a combination of a price offset & cap for `RELATIVE` order types.
+`MARKET` A/D orders execute using `MARKET` atomic orders, and offer no price
+control.
+
```

Comment:
could you add this as well here ?
 * Post-Only limit orders can also be submitted. However, when 'Await Fill' option
 * is enabled, if a sliced postonly limit order is cancelled, the algorithm will 
 * wait for the the cancelled order to fill before submitting other subsequent orders.
 * Otherwise, the algorithm just cancels the postonly orders.

Reply:
yes, good idea! lets add it once the feature is merged